network-store-server/src/main/resources/db/changelog/changesets/regulationPoint_regulatingtype_20240808T110000Z.sql

```diff
@@ -0,0 +1,32 @@
+UPDATE regulationpoint r1
```

Comment:
Have you tested this migration with real size db? And that it still works after migration? Migration times are OK?

Reply:
I think it was tested